src/main/java/com/infamous/combat_next/mixin/ItemInHandRendererMixin.java

```diff
@@ -14,8 +19,22 @@ public class ItemInHandRendererMixin {
     private boolean playReequipAnimation(boolean original){
         if(!original){
             //noinspection ConstantConditions
-            return !CombatUtil.isAttackAvailable(Minecraft.getInstance().player, 1.0F);
+            return !CombatUtil.isSupercharged(Minecraft.getInstance().player, 1.0F);
         }
         return true;
     }
+    @Redirect(method = "tick", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/player/LocalPlayer;getAttackStrengthScale(F)F"))
```

Comment:
Can't this be done with a WrapOperation instead of a Redirect?

Reply:
no????? the supercharged method works differently, just wrapping it with something isn't going to help

Reply:
fixed